qcodes/instrument_drivers/QuTech/IVVI.py

```diff
@@ -260,17 +338,15 @@ def read(self, message_len=None):
 
         while bytes_in_buffer < message_len:
             t1 = time.time()
-            time.sleep(.05)
+            time.sleep(.025)
```

Comment:
I already changed this but haven't been able to commit it to the new repo yet, both the 'set dac' sleep call and this one for waiting for the buffer are manual parameters.